students/KyleBarry/session06/test_mailroom_04.py

```diff
@@ -0,0 +1,39 @@
+import mailroom_04 as mr
+
+
+def test_make_list():
+    result = mr.make_list()
+
+    assert len(result) == len(mr.donors)
+
+
+def test_add_donation():
+    result = mr.add_donation("Tara Towers", 9999)
+
+    assert result[-1] == 9999
+
+
+def test_add_donation_new():
+    result = mr.add_donation_new("Jane Janeson", 1000)
+
+    assert result in mr.donors.values()
```

Comment:
careful about tests (and code)  like this -- they are closely tied to the data structure you are using.

Do you have a "find_donor" or similar function? You should.